app/views/shared/check_answers/_proceeding_details.html.erb

```diff
@@ -4,9 +4,12 @@
     <h2 class="govuk-heading-m"><%= proceeding.meaning %> proceeding details</h2>
   </div>
   <div class="govuk-grid-column-one-quarter">
-    <p><%= govuk_link_to t("providers.check_provider_answers.shared.change"),
-                         providers_legal_aid_application_client_involvement_type_path(@legal_aid_application, proceeding),
-                         class: "govuk-link-right" %></p>
+    <%= govuk_link_to(
+          t("generic.change"),
+          providers_legal_aid_application_client_involvement_type_path(@legal_aid_application, proceeding),
+          visually_hidden_suffix: "#{proceeding.meaning} proceeding details",
```

Comment:
Could we move this suffix value to the relevant locales file?

Reply:
Yes, but in addition to the 4 instances of `visually_hidden_suffix` there are also 53 instances of `visually_hidden_text` which already existed in the repo. Would you also suggest moving that to the relevant locales file?

Reply:
Apologies, I meant moving `proceeding details` to locales, and only for instances in this pull request.

Reply:
Thanks for this suggestion @naseberry. I have made these changes now.